### DISCUSSION TITLE
Can't connect using SMTP credetials

### DISCUSSION THREAD

**Discussion by creationshop** [postalserver/postal#2322] [Help with using Postal]
Hi everyone, 

first, thanks to postal for what looks like a great app. 

I'm trying to use smtp authentication inside laravel and it's failing. the error message is:

development.ERROR: Expected response code "250" but got code "530", with message "530 From/Sender name is not valid".

the username is given at the app like this;
user/name

Apologies if this topic is covered, i didn't see anything and have looked.

thanks!!

**Comment by willpower232** [comment]
The username is not validated at all by Postal. The error means that the From Address in the email you are trying to send uses a domain which has not been added to the Postal server where you got the credential.

**Reply by creationshop** [reply]
I had added the domain, what can I check now?

**Comment by creationshop** [comment]
Can i disable that validation? I've verified the domain is added to the server.

**Reply by willpower232** [reply]
if you've added the domain to a server in the Postal UI, you should be able to use an SMTP credential from the same area.
